tests/integration/relations/opensearch_provider/test_opensearch_provider.py

```diff
@@ -31,11 +31,12 @@
 SECONDARY_CLIENT_APP_NAME = "secondary-application"
 TLS_CERTIFICATES_APP_NAME = "tls-certificates-operator"
 ALL_APPS = [OPENSEARCH_APP_NAME, TLS_CERTIFICATES_APP_NAME, CLIENT_APP_NAME]
-FIRST_DATABASE_RELATION_NAME = "first-database"
+FIRST_DATABASE_RELATION_NAME = "first-index"
 
 NUM_UNITS = len(UNIT_IDS)
 
 
+@pytest.mark.dev
```

Comment:
Do we plan on removing this marker in favor of `client_relation` below?

Reply:
It's just something I've been using while developing, I should remove it before merge.